helpdesk-bot/src/main/java/org/symphonyoss/symphony/bots/helpdesk/bot/ticket/AcceptTicketService.java

```diff
@@ -65,6 +65,7 @@ protected TicketResponse execute(Ticket ticket, SymUser agent) {
       try {
         updateMembership(agent.getId());
         addAgentToServiceStream(ticket, agent.getId());
+        sendMessageWithShowHistoryFalse(ticket, agent.getId());
```

Comment:
You should rename this method to 'sendTicketHistory'. You might also declare a variable to hold the agent id.

Reply:
Done.